pkgs/by-name/br/bruno-cli/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildNpmPackage,
+  bruno,
+  pkg-config,
+}:
+
+buildNpmPackage {
+  pname = "bruno-cli";
+  inherit (bruno) version src npmDepsHash;
+
+  npmWorkspace = "packages/bruno-cli";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  dontNpmBuild = true;
+  postBuild = ''
+    npm run build --workspace=packages/bruno-common
+    npm run build --workspace=packages/bruno-query
+    npm run build --workspace=packages/bruno-graphql-docs
+
+    npm run sandbox:bundle-libraries --workspace=packages/bruno-js
+  '';
+
+  preFixup = ''
+    cp -r packages $out/lib/node_modules/usebruno
```

Comment:
Maybe we should put the node modules into `$out/lib/bruno-cli/node_modules`? It's not likely to make a conflict, but seems safer just in case we'd something like this to `bruno`.

I'm using a few packages that install their files to a to eg. `$out/lib/LICENSE` and it causes conflicts by deafult with home-manager.